lib/themes/dosomething/paraneue_dosomething/js/campaign/GalleryController.js

```diff
@@ -0,0 +1,79 @@
+import setting from '../utilities/Setting';
+import ApiClient from '../utilities/ApiClient';
+import { sortCampaignsByRelevance } from '../campaign/helpers';
+
+class GalleryController {
+  constructor() {
+    this.campaignsToExclude = setting('dsUser.activity.signups', []);
+    this.campaignsApiPage = 1;
+    this.apiClient = new ApiClient('v1');
+    this.retrievedCampaigns = [];
+  }
+
+  filter(campaigns, comparison) {
+    return campaigns.filter(campaign => comparison.indexOf(parseInt(campaign.id)) == -1);
+  }
+
+  /**
+   * Grab campaigns & alter based on given args.
+   * @param {Object} args containing the following optional parameters
+   *   - {boolean} isStaffPick - Defaults to false.
+   *   - {int} page - Defaults to 1.
+   *   - {boolean} removeDuplicateResults
+   *   - {boolean} sortByRelevane
+   *   - {boolean} removeSignups
+   * @return {Promise}
+   */
+  getCampaigns(args) {
+    const query = {
+      staff_pick: args['isStaffPick'] ? +args['isStaffPick'] : 0, // API requires 1/0 instead of true/false
```

Comment:
I don't believe this is true. You should be able to pass `true` or `false` instead of `1` or `0`.

```
http://staging.beta.dosomething.org/api/v1/campaigns?staff_pick=true
```
works just fine.